plugins/experimental/esi/esi.cc

```diff
@@ -61,6 +61,7 @@ struct OptionInfo {
 };
 
 static HandlerManager *gHandlerManager = NULL;
+static Utils::HeaderValueList gWhitelistCookies;
```

Comment:
I'm not familiar with the ESI plugin, but there's only one "config" ever? It sort of feels that the whitelist ought to be associated with the handler (config) no? Even if not supported now, maybe later you would want to?